packages/client-core/src/admin/components/locations/AddEditLocationModal.tsx

```diff
@@ -191,39 +205,37 @@ export default function AddEditLocationModal(props: {
       unPublishLoading.set(false)
     }
   }
+  const anchorEvent = useHookstate<null | React.MouseEvent<HTMLElement>>(null)
 
   return (
-    <div className="relative z-50 w-[50vw] bg-surface-1">
-      <div className="relative rounded-lg shadow">
-        <ModalHeader
-          onClose={PopoverState.hidePopupover}
-          title={location?.id ? t('editor:toolbar.publishLocation.update') : t('editor:toolbar.publishLocation.create')}
-        />
-        <div className="h-fit max-h-[60vh] w-full overflow-y-auto px-10 py-6">
+    <div className="relative z-50 w-[60vw] bg-surface-1 px-8 pt-6">
+      <div className="relative rounded-lg py-2">
+        <div className="flex justify-between pb-6">
+          <h2 className="text-xl">
```

Comment:
lets skip using h2 because heading tags require their siblings to be present 
lets use simple `span`
and they should not be used in a modal where the content is dynamic